tutorials/physics/interpolation/advanced_physics_interpolation.rst

```diff
@@ -0,0 +1,173 @@
+
+Typical example
+^^^^^^^^^^^^^^^
+
+A typical example of a custom approach is to use the ``look_at`` function in the
+Camera every frame in ``_process()`` to look at a target node (such as the player).
+
+But there is a problem. If we use the traditional ``get_global_transform()`` on a
+Camera "target" Node, this transform will only focus the Camera on the target *at
+the current physics tick*. This is *not* what we want, as the Camera will jump
+about on each physics tick as the target moves. Even though the Camera may be
+updated each frame, this does not help give smooth motion if the *target* is only
+changing each physics tick.
+
+get_global_transform_interpolated()
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+What we really want to focus the Camera on, is not the position of the target on
+the physics tick, but the *interpolated* position, i.e. the position at which the
+target will be rendered.
+
+We can do this using the :ref:`Spatial.get_global_transform_interpolated<class_Node3D_method_get_global_transform_interpolated>`
+function. This acts exactly like getting :ref:`Spatial.global_transform<class_Node3D_property_global_transform>`
+but it gives you the *interpolated* transform (during a ``_process()`` call).
+
+.. important:: ``get_global_transform_interpolated()`` should only be used once or
+	           twice for special cases such as Cameras. It should **not** be used
+               all over the place in your code (both for performance reasons, and
+			   to give correct gameplay).
+
+.. note:: Aside from exceptions like the Camera, in most cases, your game logic
+	      should be in ``_physics_process()``. In game logic you should be calling
+          ``get_global_transform()`` or ``get_transform()``, which will give the
+		  current physics transform (in global or local space respectively), which
+		  is usually what you will want for gameplay code.
+
+Example manual Camera script
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Here is an example of a simple fixed Camera which follows an interpolated target:
+
+.. code-block:: python
+
+	extends Camera
+		
+	# Node that the camera will follow
+	var _target
+		
+	# We will smoothly lerp to follow the target
+	# rather than follow exactly
+	var _target_pos : Vector3 = Vector3()
+		
+	func _ready() -> void:
+		# Find the target node
+		_target = get_node("../Player")
+		
+		# Turn off automatic physics interpolation for the Camera,
+		# we will be doing this manually
+		set_physics_interpolation_mode(Node.PHYSICS_INTERPOLATION_MODE_OFF)
+		
+	func _process(delta: float) -> void:
+		# Find the current interpolated transform of the target
+		var tr : Transform = _target.get_global_transform_interpolated()
+		
+		# Provide some delayed smoothed lerping towards the target position 
+		_target_pos = lerp(_target_pos, tr.origin, min(delta, 1.0))
+		
+		# Fixed camera position, but it will follow the target
+		look_at(_target_pos, Vector3(0, 1, 0))
```

Comment:
This whole section uses tabs, not spaces. Somehow it's rendering fine locally for me. You should probably still change it?

Reply:
I'll change it